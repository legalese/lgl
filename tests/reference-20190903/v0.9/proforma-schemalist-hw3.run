LGL_URI=http://localhost/api/corpsec/v0.9 lgl --version=0.9 proforma schemalist hw3
