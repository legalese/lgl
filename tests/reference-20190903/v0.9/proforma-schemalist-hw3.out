{
  "about": {
    "title": "Hello World 3",
    "templateVersion": "1.0.0",
    "prerequisites": [],
    "show_in_development": true,
    "show_in_production": true,
    "filepath": "hw3",
    "templateKey": "hw3"
  },
  "example": {
    "company": {
      "fullName": "My Unicorn Pte. Ltd.",
      "id": {
        "idtype": "UEN",
        "idvalue": "204000000X"
      },
      "nationality": "SG",
      "address": {
        "lines": [
          "71 Ayer Rajah Crescent",
          "#00-00"
        ],
        "postcode": "139951"
      }
    },
    "directors": [
      {
        "fullName": "Alice",
        "id": {
          "idtype": "NRIC",
          "idvalue": "S1111111A"
        },
        "nationality": "SG",
        "address": {
          "lines": [
            "0 Sentosa Cove"
          ],
          "postcode": "111111"
        }
      },
      {
        "fullName": "Bob",
        "id": {
          "idtype": "NRIC",
          "idvalue": "S2222222X"
        },
        "nationality": "SG",
        "address": {
          "lines": [
            "0 Changi Way",
            "#00-00"
          ],
          "postcode": "222222"
        }
      }
    ]
  }
}
